packages/@aws-cdk/aws-lambda-nodejs/package.json

```diff
@@ -78,7 +78,9 @@
     "@aws-cdk/integ-runner": "0.0.0",
     "@aws-cdk/integ-tests": "0.0.0",
     "@aws-cdk/pkglint": "0.0.0",
+    "@aws-cdk/triggers": "0.0.0",
     "@types/jest": "^27.5.2",
+    "axios": "^0.27.2",
```

Comment:
```suggestion
```

I don't think this is needed anymore

Reply:
Removed

Reply:
For some reason, the tests in `test/docker.test.js` are now failing

I can see in the logs:

```
time="2022-12-27T14:51:42.954645757Z" level=info msg="Attempting next endpoint for pull after error: denied: User: arn:aws:sts::464831875156:assumed-role/Ops-cdk-autobuild-prs-AutoBuildv2ProjectRoleC688F4-CY3ZTUL6CY8Q/AWSCodeBuild-b78b1055-51c8-4066-8449-8e563d561d94 is not authorized to perform: ecr:BatchGetImage on resource: arn:aws:ecr:us-east-1:464831875156:repository/library/esbuild because no identity-based policy allows the ecr:BatchGetImage action"
```

Reply:
Squashing commits seemed to fix it, not sure why